easybuild/tools/build_log.py

```diff
@@ -107,8 +107,9 @@ def nosupport(self, msg, ver):
 
     def error(self, msg, *args, **kwargs):
         """Print error message and raise an EasyBuildError."""
-        newMsg = "EasyBuild crashed with an error %s: %s" % (self.caller_info(), msg)
-        fancylogger.FancyLogger.error(self, newMsg, *args, **kwargs)
+        newMsg = ("EasyBuild crashed with an error %s: %s"
+                  % (self.caller_info(), (msg % args)))
+        fancylogger.FancyLogger.error(self, newMsg, **kwargs)
         if self.raiseError:
             raise EasyBuildError(newMsg)
```

Comment:
@riccardomurri: isn't this still a problem for GC3Pie?

we should look into a way of only raising when the logger instance is used inside EB